index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Trials Survey</title>
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+
+  <body>
+    <div id="container">
+      <div class="header">
+        <h1 id="title">Microverse Trials Survey</h1>
+        <p id="description"><em>This survey is to help improve the quality of the Microverse Trials for applicants</em></p>
+      </div>
+      <div class="form">
+        <form id="survey-form">
+          
+          <div>
+            <label id="name-label">Name</label>
```

Comment:
Please add a `for` attribute to your `label` tag. This helps associate the `label` with the `form` element that is being labeled

Reply:
It would be better to explain your comments politely. For example, you can use this sentence at the beginning of your comment "I think it would be better to add a `for`  attribute to your `label` tag".